src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -32,6 +32,36 @@ export const DEFAULT_API_SCOPES = [
   'member:read',
 ];
 
+// These should only be used in the case where we cannot obtain roles through
+// the members endpoint (primarily in cases where a user is admining a
+// different organization they are not a OrganizationMember of ).
+export const MEMBER_ROLES = [
+  {
```

Comment:
Should the `billing` role be available as well?

Reply:
It's not in sentry 🤷‍♂

Honestly this should turn into an endpoint or soemthing to get all roles, for things to be more 'correct'